src/index.js

```diff
@@ -2,7 +2,8 @@
 const generateDocFromSchema = require("./lib/generator");
 const path = require("path");
 const os = require("os");
-const GroupInfo = require("./lib/group-info");
+const { pluginConfigs } = require("./utils/helpers/other.js");
```

Comment:
`pluginConfigs[]` should not be exposed, see comment regarding `pluginConfigs.push({ ...DEFAULT_OPTIONS, ...opts });`

Reply:
Addressed in 9b550e2.